examples/basics/data_rows.ipynb

```diff
@@ -162,7 +162,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    
</pre><p>nit: extra space not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done</p>